microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -93,31 +111,41 @@ If you need to access more detailed information, use the [getHighEntropyValues()
+Combining User-Agent Client Hints with [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection] is an effective way to deliver compatible web content. Microsoft recommends using this pattern to:
+* Improve code maintainability.
+* Reduce code fragility.  
+* Reduce code breakage from changes to the User-Agent String.
 
-*   A fix for a Chromium bug in a later version must be avoided and the affected browsers are difficult to detect outside of a verification of the brand and version.  
-    
-Instead, you should verify the `Chromium` brand, which ensures your detection applies to all affected Chromium-based browsers.  
+If you need to check for a Chromium-based browser, Microsoft recommends detecting `Chromium`; the engine that powers both Microsoft Edge and Google Chrome.
 
+Use this method to verify the `Chromium` brand and apply detection to all affected Chromium-based browsers.
 
 ```javascript
 function isChromium() {
     for (brand_version_pair in navigator.userAgentData.brands) {
-        if (brand_version_pair.brand == "Chromium") {
+        if (brand_version_pair.brand == "Chromium"){
             return true;
         }
     }
     return false;
 }
-```  
+```
+
+Display orders of brand names change over time. Avoid hard-coding checks for brands at specific indices.
+
+If you cannot use [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection], don't use a hardcoded list of known Chromium-based browsers for verification. Examples of hardcoded browser names include `Microsoft Edge` and `Google Chrome`. [Feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection] may not be available because a fix for a Chromium bug in later versions must be avoided and the affected browsers are difficult to detect.
+
+## User-Agent strings
 
-## User agent string  
+User-Agent strings are outdated and have a long history of causing website compatibility issues.
 
-Wherever possible, Microsoft recommends you minimize your use of Microsoft Edge browser detection logic based on the user agent string.  If you have a good reason to detect the user agent, the Microsoft Edge team recommends you use [User-Agent Client Hints](#user-agent-client-hints) as the primary detection logic.  [User-Agent Client Hints](#user-agent-client-hints) also reduces the required number of parsed strings.  However, for legacy reference, the following format is used for the User-Agent string.  
+Wherever possible, Microsoft recommends minimizing use of Microsoft Edge browser detection logic based on the user-agent string. If you have a good reason to detect the user agent, the Microsoft Edge team recommends using [User-Agent Client Hints](#user-agent-client-hints) as the primary detection logic. [User-Agent Client Hints](#user-agent-client-hints) also reduces the required number of parsed strings.

```

Comment:
nitpick: It feels like we're saying "user-agent string" a lot here. Maybe we can update this phrase:

> If you have a good reason to detect the user agent,

To be something like:

> If you have a good reason to detect the browser,

Reply:
nitpick: The phrasing at the end:

> also reduces the required number of parsed strings.

reads a bit more accurately if it's something like:

> also reduces the complexity in browser detection code.

I say this because I don't know if it's accurate to say that UA CH involves significantly less parsing in general, but it's definitely simpler to implement.

Reply:
Done and done.